pages.nl/common/amass-enum.md

```diff
@@ -0,0 +1,24 @@
+# amass enum
+
+> Vind subdomeinen van een domein.
+> Meer informatie: <https://github.com/OWASP/Amass/blob/master/doc/user_guide.md#the-enum-subcommand>.
+
+- Vind passief subdomeinen van een domein:
```

Comment:
Are you sure that it is `passieve`? I think it is in this sentence a passive way to search something instead of passive subdomains.

The english sentence is: `Passively find subdomains of a domain:`

So maybe `Vind, passief, subdomeinen van een domein`?